### PR TITLE
Mailposter

### PR DESCRIPTION
**Pull request by Kr8art**
## About the PR
Adds a new mail-themed poster. First PR, please don't hurt me. 

## Why / Balance
Flavour, wanted to learn how to do a PR, no balance implications (also Lyndo told me to do this). 

## Technical details
- Added a new folder with the meta/image in the DeltaV textures folder
- Created new entity in the poster.yml
- Added the poster id to the random poster spawner list (with comment) 

## Media
image

## Requirements

- [X] I have tested all added content and changes.
- [X] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes
n/a

**Changelog**
:cl:
- add: New mail-themed poster!

### CODE REVIEW

**Comment by dvir001** [comment]
MAIL UP

**Comment by Lyndomen** [comment]
This looks great, nice job!

**Comment by MilonPL** [comment]
BASED

File: Resources/Textures/DeltaV/Structures/Wallmounts/Posters/Mailposter.rsi/meta.json
Comment (MilonPL) [inline Resources/Textures/DeltaV/Structures/Wallmounts/Posters/Mailposter.rsi/meta.json:1]: it should be lowercase mailposter.rsi

File: Resources/Prototypes/DeltaV/Entities/Structures/Wallmounts/Signs/posters.yml [hunk 1]
```diff
@@ -17,3 +17,13 @@
...
     state: woodygotwood
+
+- type: entity
+  parent: PosterBase
+  id: PosterLegitMail
+  name: "Only You Can Prevent Expired Mail"
+  description: "A propaganda poster that subtly encourages crew to open their mail. Brought to you by the Ministries Aiding Interstellar Logistics (M.A.I.L.)"
+  components:
+  - type: Sprite
+    sprite: DeltaV/Structures/Wallmounts/Posters/Mailposter.rsi
```
Comment (MilonPL) [inline Resources/Prototypes/DeltaV/Entities/Structures/Wallmounts/Signs/posters.yml]: lowercase mailposter

**Review by MilonPL — CHANGES_REQUESTED** [review]
minor nitpick, looks good otherwise

**Comment by Kr8art** [comment]
Removed capitalization in foldername and poster.yml

**Comment by Tomce795** [comment]
Don't do a me and forget to request a review for the changes again. Otherwise Milon with forgor :blunt: Click the extra options on the "MilonPL requested changes" thingy and select "request review" or something along those lines.

**Comment by Kr8art** [comment]
> Don't do a me and forget to request a review for the changes again. Otherwise Milon with forgor :blunt: Click the extra options on the "MilonPL requested changes" thingy and select "request review" or something along those lines.

Yeah, I totally knew I had to do that 😅

**Review by MilonPL — APPROVED** [review]

**Pull request merged**